functions/backend/perplexity_reddit_scrape.py

```diff
@@ -0,0 +1,50 @@
+import json
+from scrapereddit import scrape_reddit_reviews
+
+author = "author"
+content = "content"
+created = "created"
+rating = "rating"
+show = "show"
+source = "source"
+title = "title"
+type = "type"
+scraped_ratings_dict = {}
+
+all_urls: dict = {
```

Comment:
Can you add the code for perplexity here? So we can get 5 urls for any tv show name passed in